src/config.h.cmake

```diff
@@ -4,7 +4,7 @@
 #cmakedefine BGDWIN32
 
 /* Whether to support gd image formats */
-#cmakedefine01 ENABLE_GD_FORMATS
+#define ENABLE_GD_FORMATS @ENABLE_GD_FORMATS@
```

Comment:
guessing you didn't mean to make this change

Reply:
yeah, I thought I removed this file from the commit, as it's autogenerated... I wanted to ask why it's included in the repo?

Reply:
i think this is the only file that one needs for using cmake, so we make a concession in that regard